tap_github/schemas/comments.json

```diff
@@ -93,6 +99,199 @@
       "type": ["null", "string"],
       "format": "date-time"
     },
+    "performed_via_github_app": {
```

Comment:
It is used twice, move this in the common folder.

Reply:
Added a shard schema for the same